task8/script.js

```diff
@@ -0,0 +1,61 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+// ローディング画像を表示
```

Comment:
Let's turn off the comment out, because we can see what it is by reading the function.

Reply:
You should delete unnecessary comment outs.

commit: 48729d2